ansible/roles/vpn_mgt/templates/bird.conf.j2

```diff
@@ -26,6 +26,18 @@ protocol static {
     ipv4;                   # Again, IPv4 channel with default options
 }
 
+protocol bfd {
+{% for wg_config in wireguard_configs %}
+{% if wg_config.BFD_ENABLE is defined %}
```

Comment:
When BFD_ENABLE is set to `no`, wouldn’t the BFD config still be used here?

Reply:
good question - the answer is no. If BFD is set to NO in the ospf section of the bird.conf, the BFD protocol section is ignored. I thought about making it so the ansible variable only enabled BFD, but then I thought of scenarios where you may want to turn off BFD at the server level, but keep your previous configuration. My pull allows for this.

Reply:
I suppose we can make BFD_ENABLE truely a boolean... if its defined, BFD is set to yes in the ospf protocol section. If BFD_ENABLE is not set, by default it is no.... so the other BFD settings are ignored, but maintained, in the bird.conf file

Reply:
```suggestion
{% if wg_config.BFD_ENABLE is defined and wg_config.BFD_ENABLE %}
```